### ISSUE TITLE
Can't run on session-less environment (non-systemd).

### ISSUE THREAD

**Issue by ahmadraniri** [bleachbit/bleachbit#1618]
**Describe the bug**
Bleachbit can't run on session-less environment (checked by `loginctl`), still need display manager or session creator in order to run (apparently, using display manager can't solve the issue).

**To Reproduce**
1. Launch wayland compositor / x11 window manager directly via tty (not from display manager)
1. Open terminal
1. Running `bleachbit`.
1. Getting this output : 

```
Traceback (most recent call last):
  File "/usr/local/bin/bleachbit", line 40, in <module>
    bleachbit.Unix.is_display_protocol_wayland_and_root_not_allowed()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'bleachbit.Unix' has no attribute 'is_display_protocol_wayland_and_root_not_allowed'
```

The error happens (every time).

**Expected behavior**
Bleachbit will run although we have no session (like bleachbit 4.4.2).

**Desktop (please complete the following information):**
 - OS: Alpine Linux.
 - Wayland compositor: swaywm
 - X11 window manager: cwm.  
 - BleachBit version: Latest master.

**Comment by ahmadraniri** [comment]
Comment some lines on `bleachbit.py` and `bleachbit/Unix.py` seems fix it. Bleachbit can be launched.

**bleachbit.py**
```
# if 'posix' == os.name:
#     if os.path.isdir('/usr/share/bleachbit'):
#         # This path contains bleachbit/{C,G}LI.py .  This section is
#         # unnecessary if installing BleachBit in site-packages.
#         sys.path.append('/usr/share/')
# 
#     # The two imports from bleachbit must come after sys.path.append(..)
#     import bleachbit.Unix
#     from bleachbit import _
# 
#     if (
#         bleachbit.Unix.is_display_protocol_wayland_and_root_not_allowed()
#     ):
#         print(_('To run a GUI application on Wayland with root, allow access with this command:\n'
#               'xhost si:localuser:root\n'
#                 'See more about xhost at https://docs.bleachbit.org/doc/frequently-asked-questions.html'))
#         sys.exit(1)
```

**bleachbit/Unix.py**
```
# def is_display_protocol_wayland_and_root_not_allowed():
#     try:
#         is_wayland = bleachbit.Unix.is_unix_display_protocol_wayland()
#     except Exception as e:
#         logger.exception(e)
#         return False
#     return (
#         is_wayland and
#         os.environ['USER'] == 'root' and
#         bleachbit.Unix.root_is_not_allowed_to_X_session()
#     )
```

**Comment by ahmadraniri** [comment]
Using display manager doesn't fix the issue, apparently.

**Comment by ahmadraniri** [comment]
So, I try other alternative, I replace some lines in `is_linux_display_protocol_wayland` function. It works apparently, I don't know much about python though.

```
def is_linux_display_protocol_wayland():
    assert(sys.platform.startswith('linux'))
    result = General.run_external(['echo', '$XDG_SESSION_TYPE'])
    return 'wayland' in result[1].lower()
```

**Comment by ahmadraniri** [comment]
Fixed and solved, turns out the older version (4.4.x) was still around at `/usr/lib/python3.11/site-packages/bleachbit-4.4.2-py3.11.egg`. Deleting directory fixs.
